google/cloud/bigtable/tests/run_integration_tests_emulator.sh

```diff
@@ -64,3 +64,7 @@ echo "Running TableAdmin async integration test."
 echo
 echo "Running Table::Async* integration test."
 ./data_async_integration_test "${PROJECT_ID}" "data-noex-async"
+
```

Comment:
Can you remove the commented-out code?  If you think we need a reminder to fix this in the future just open a bug.

Reply:
created #1338 for tracking.